resources/assets/js/app/views/votes/UsersVoteItemView.js

```diff
@@ -0,0 +1,22 @@
+var Marionette = require('backbone.marionette');
+
+module.exports = Marionette.ItemView.extend({
+    template: 'voteItem',
+    className: 'row post-item',
+    tagName: 'div',
+    serializeData: function () {
+        var tempmeta = this.model.getMeta();
+        console.log(tempmeta);
+        var id = this.model.get('id');
+        return {
+            model: this.model.toJSON(),
```

Comment:
Is that model of what? Please use understandable option name (don't forget to replace it in template)